src/Nest/Mapping/Types/Specialized/Attachment/AttachmentProperty.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace Nest
 {
+	[Obsolete("Removed in 6.0.")]
 	[JsonObject(MemberSerialization.OptIn)]
 	public interface IAttachmentProperty : IDocValuesProperty
```

Comment:
`IAttachmentProperty` can be removed altogether in 6.x - it's used for the mapper-attachment plugin in 5.x, which is superseded by the ingest-attachment plugin.